controller.py

```diff
@@ -66,8 +69,39 @@ def start_new_game():
   (and continuation) of a game of Bites.
   """
   players = prepare_list_of_players()
-  play_bites = Bites(ANTS, TOKENS_FOR_TRAIL, players)
+  anthill_order = choose_anthill_rule()
+  play_bites = Bites(ANTS, TOKENS_FOR_TRAIL, players, anthill_order)
   play_bites.play_full_game()
 
+def choose_anthill_rule(anthill_card_dict=ANTHILL_CARD_DICT):
```

Comment:
Here you've given as a default argument, something that is imported from another file. I've not seen this before and I suspect it's not good practice. You need the `anthill_card_dict` as an argument in this function at all. I suggest you remove it and just the `ANTHILL_CARD_DICT` constant directly inside the function.

Reply:
An alternative is to have the argument `anthill_card_dict=None` and then inside the function:
```
if anthill_card_dict is None:
    anthill_card_dict = ANTHILL_CARD_DICT
```
This way you can still specify the `anthill_card_dict` in the unit tests.

Reply:
I'm now leaning towards the latter option of `anthill_card_dict=None`